sklearn/ensemble/bagging.py

```diff
@@ -267,6 +267,9 @@ def _fit(self, X, y, max_samples, sample_weight=None):
         max_samples : int or float, optional (default=None)
             Argument to use instead of self.max_samples.
 
+        max_depth : int, optional (default=None)
+            Maximum depth of the base estimator.
```

Comment:
you should say 
  ```
Argument to use instead of the one initially passed to the base estimator.
This is supported only if the base estimator has a max_depth parameter.
```